Add Hiragana and Katagana Kanji characters to support Japanese translation

PhracturedBlue opened the pull request:
Add new Kanji characters (0x3040 - 0x30ff) to support new Japanese translation for both 12normal and 15normal fonts

howard0su commented:
Could not locate character U-5909
Could not locate character U-7d9a
Could not locate character U-52b9
Could not locate character U-5358

I got some warnings when I am playing with this change on JP.

PhracturedBlue commented:
I think this is the full list of characters in our translations that don't have font characters (need to sanity check some of those...a couple don't make any sense)
```
Missing: 0x7f:  (7f)
Missing: 0x98:  (c2 98)
Missing: 0x190: Ɛ (c6 90)
Missing: 0x19f: Ɵ (c6 9f)
Missing: 0x1c4: Ǆ (c7 84)
Missing: 0x24d: ɍ (c9 8d)
Missing: 0x2b9: ʹ (ca b9)
Missing: 0x318: ̘ (cc 98)
Missing: 0x6bd: ڽ (da bd)
Missing: 0x7ce: ߎ (df 8e)
Missing: 0x3001: 、 (e3 80 81)
Missing: 0x3002: 。 (e3 80 82)
Missing: 0x5024: 値 (e5 80 a4)
Missing: 0x52b9: 効 (e5 8a b9)
Missing: 0x5358: 単 (e5 8d 98)
Missing: 0x5727: 圧 (e5 9c a7)
Missing: 0x5909: 変 (e5 a4 89)
Missing: 0x5b9f: 実 (e5 ae 9f)
Missing: 0x5bfe: 対 (e5 af be)
Missing: 0x623b: 戻 (e6 88 bb)
Missing: 0x629e: 択 (e6 8a 9e)
Missing: 0x62e1: 拡 (e6 8b a1)
Missing: 0x633f: 挿 (e6 8c bf)
Missing: 0x6442: 摂 (e6 91 82)
Missing: 0x691c: 検 (e6 a4 9c)
Missing: 0x7d4c: 経 (e7 b5 8c)
Missing: 0x7d76: 絶 (e7 b5 b6)
Missing: 0x7d9a: 続 (e7 b6 9a)
Missing: 0x7dd1: 緑 (e7 b7 91)
Missing: 0x8ee2: 転 (e8 bb a2)
Missing: 0x9045: 遅 (e9 81 85)
Missing: 0xfeff: ﻿ (ef bb bf)
Missing: 0xff01: ！ (ef bc 81)
Missing: 0xff1a: ： (ef bc 9a)
Missing: 0xff1f: ？ (ef bc 9f)
Missing: 0xfffd: � (ef bf bd)
```

PhracturedBlue commented:
I have cleaned up the translations to use Latin punctuation for asian languages.  I also added the missing characters such that the 12 and 15 fonts contain all characters used in all translations
